js_modules/dagit/packages/core/src/instance/BackfillTable.mocks.ts

```diff
@@ -0,0 +1,419 @@
+    query: SINGLE_BACKFILL_STATUS_COUNTS_QUERY,
+    variables: {
+      backfillId: 'tclwoggv',
+    },
+  },
+  result: {
+    data: {
+      __typename: 'DagitQuery',
+      partitionBackfillOrError: {
+        backfillId: 'tclwoggv',
+        partitionStatusCounts: [
+          {runStatus: RunStatus.NOT_STARTED, count: 6524, __typename: 'PartitionStatusCounts'},
+        ],
+        __typename: 'PartitionBackfill',
+      },
+    },
+  },
+};
+
+export const BackfillTableFragmentFailedError: BackfillTableFragment = {
+  backfillId: 'sjqzcfhe',
+  status: BulkActionStatus.FAILED,
+  numPartitions: 100,
+  timestamp: 1674774274.343382,
+  partitionSetName: null,
+  partitionSet: null,
+  error: {
+    __typename: 'PythonError',
+    message:
+      'dagster._core.errors.DagsterLaunchFailedError: Tried to start a run on a server after telling it to shut down\n',
+    stack: [
+      '  File "/Users/bengotow/Work/F376/Projects/dagster/python_modules/dagster/dagster/_daemon/backfill.py", line 34, in execute_backfill_iteration\n    yield from execute_asset_backfill_iteration(backfill, workspace, instance)\n',
+      '  File "/Users/bengotow/Work/F376/Projects/dagster/python_modules/dagster/dagster/_core/execution/asset_backfill.py", line 193, in execute_asset_backfill_iteration\n    submit_run_request(\n',
+      '  File "/Users/bengotow/Work/F376/Projects/dagster/python_modules/dagster/dagster/_core/execution/asset_backfill.py", line 265, in submit_run_request\n    instance.submit_run(run.run_id, workspace)\n',
+      '  File "/Users/bengotow/Work/F376/Projects/dagster/python_modules/dagster/dagster/_core/instance/__init__.py", line 1926, in submit_run\n    submitted_run = self._run_coordinator.submit_run(\n',
+      '  File "/Users/bengotow/Work/F376/Projects/dagster/python_modules/dagster/dagster/_core/run_coordinator/default_run_coordinator.py", line 34, in submit_run\n    self._instance.launch_run(pipeline_run.run_id, context.workspace)\n',
+      '  File "/Users/bengotow/Work/F376/Projects/dagster/python_modules/dagster/dagster/_core/instance/__init__.py", line 1979, in launch_run\n    self.run_launcher.launch_run(LaunchRunContext(pipeline_run=run, workspace=workspace))\n',
+      '  File "/Users/bengotow/Work/F376/Projects/dagster/python_modules/dagster/dagster/_core/launcher/default_run_launcher.py", line 121, in launch_run\n    DefaultRunLauncher.launch_run_from_grpc_client(\n',
+      '  File "/Users/bengotow/Work/F376/Projects/dagster/python_modules/dagster/dagster/_core/launcher/default_run_launcher.py", line 89, in launch_run_from_grpc_client\n    raise (\n',
+    ],
+    errorChain: [],
+  },
+  numCancelable: 0,
+  partitionNames: buildTimePartitionNames(new Date('2020-01-01'), 100),
+  assetSelection: [
+    {
+      path: ['multipartitioned_asset'],
+      __typename: 'AssetKey',
+    },
+  ],
+  __typename: 'PartitionBackfill',
+};
+
+export const BackfillTableFragmentFailedErrorStatus: MockedResponse<SingleBackfillQuery> = {
```

Comment:
It took about half an hour to create these files by copy-pasting requests / responses out of the Chrome network tab and cutting them down to be representative examples only.

One annoying part is that TypeScript expects it's enums for things like RunStatus and won't accept the string "COMPLETED" instead of `RunStatus.Completed`, so it takes a bit of manual tweaking...